operations/agent-flow-mixin/alerts/clustering.libsonnet

```diff
@@ -41,22 +33,7 @@ alert.newGroup(
         cluster_node_gossip_health_score > 0
       |||,
       'Cluster node is reporting a health score > 0.',
-      '5m',
-    ),
-
-    // Lamport clock of a node is not progressing at all.
-    //
-    // This only checks for nodes that have peers other than themselves; nodes
-    // with no external peers will not increase their lamport time because
-    // there is no cluster networking traffic.
-    alert.newRule(
-      'ClusterLamportClockStuck',
-      |||
-        sum by (cluster, namespace, instance) (rate(cluster_node_lamport_time[2m])) == 0
-        and on (cluster, namespace, instance) (cluster_node_peers > 1)
-      |||,
-      "Cluster nodes's lamport clocks is not progressing.",
-      '5m',
```

Comment:
I think we can keep this alert; this does not depend on peer timing but is purely for a specific node. WDYT?

Reply:
It would still be a cause and not a symptom. If the clock is stuck, we will eventually get a cluster transition and the other alert will fire. 

If we try to predict all possible causes, we'll likely fail and we'll also have a ton of alerts and they will misfire frequently, so I'm feeling quite strongly against alerts on causes.

Reply:
Ok, let's move ahead with this and see how it plays out.